app/(auth)/(tabs)/index.tsx

```
@@ -26,38 +25,38 @@ export default function HomeScreen() {
         <HelloWave />
       </ThemedView>
       <ThemedView style={styles.stepContainer}>
-        <ThemedText type="subtitle">Step 1: Try it</ThemedText>
-        <ThemedText>
-          Edit <ThemedText type="defaultSemiBold">app/(tabs)/index.tsx</ThemedText> to see changes.
+        <Text type="subtitle">Step 1: Try it</Text>
```

ESLint

[error] 'Text' is not defined.

(react/jsx-no-undef)